src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -517,6 +517,9 @@
         - name: --aks-custom-headers
           type: string
           short-summary: Send custom headers. When specified, format should be Key1=Value1,Key2=Value2
+        - name: --max-surge
+          type: string
+          short-summary: int (5) or percent (33%)  indidcating the extra nodes to use to speed upgrade
 """
```

Comment:
typo "indidcating", and summary could be arranged for better format. like
short-summary: extra nodes used to speed upgrade. When specified,  number or percent needed, eg: 5 or 33%

Reply:
stole that directly. :)